### PR TITLE
Add fields to the session model

### PR DESCRIPTION
**Pull request by annoyingpr**
### Notion ticket link

[Add fields to the session model](https://www.notion.so/Add-fields-to-the-session-model-596f754e57054fb69f6343e96ef0776e)

### Implementation description

- Added `fields` json column (followed [Daniel's PR](https://github.com/uwblueprint/project-read-backend/pull/20) quite closely)
- Added `validate_fields` validator
- Added tests for validator 

### Steps to test

1. `python manage.py test`

### What should reviewers focus on?

Updated field object to be the following after our discussion:
```
[
    // field IDs
    1, 2, 3, 4, 5
]
```

### Checklist

- [x] My PR name is descriptive and in imperative tense
- [x] I have run the linter
- [x] I have requested a review from the PL, and/or other devs who have background knowledge on this PR or who will be building on top of this PR

### CODE REVIEW

**Comment by danielchen115** [comment]
I'm fine with the current structure :) verbosity never hurt anybody

File: enrolments/tests/test_validators.py [hunk 1]
```diff
@@ -12,7 +13,39 @@ def setUp(self):
                 Student(first_name="Buckets", last_name="Jr", role="Child"),
             ]
         )
+        Field.objects.bulk_create(
```
Comment (danielchen115) [inline enrolments/tests/test_validators.py:16]: insightful questions 🔬

**Review by danielchen115 — APPROVED** [review]
lgtm!

File: enrolments/tests/test_validators.py (hunk 1 above)
Comment (Tayef-Shah) [reply enrolments/tests/test_validators.py:16]: I agree, very important questions everyone should be asking themselves

File: enrolments/validators.py [hunk 2]
```diff
@@ -48,6 +48,34 @@ def validate_attendance(classObj):
             )
 
 
+def validate_fields(session_obj):
```
Comment (brittanylau) [inline enrolments/validators.py]: is the input for this function a session or a field object?

File: enrolments/validators.py [hunk 3]
```diff
@@ -48,6 +48,34 @@ def validate_attendance(classObj):
...
+    Validates against the fields column for the Session model.
+    The expected structure is:
+    [
+        {
+            "fields": [
+                // field IDs
+                1, 2, 3, 4, 5
+            ]
+        },
+    ]
```
Comment (brittanylau) [inline enrolments/validators.py]: is there a reason why we need to have an outer array? is it valid to have multiple "field" objects this array?

**Review by brittanylau — COMMENTED** [review]
looks good overall, a few questions!

I'm not opposed to having an array of field IDs (the second approach mentioned), because when we add this to the serializer, the response object will have the "fields" key anyway

File: enrolments/validators.py (hunk 3 above)
Comment (annoyingpr) [reply enrolments/validators.py]: now that i take a second look, i don't think is necessary! updating so it's just the JSON object

File: enrolments/validators.py (hunk 2 above)
Comment (annoyingpr) [reply enrolments/validators.py]: it's a field object, will update the naming accordingly!

**Review by brittanylau — APPROVED** [review]
looks gouda 🧀  grate work!

**Pull request merged**